pyoverkiz/client.py

```diff
@@ -121,6 +127,18 @@ def __init__(
 
         self.session = session if session else ClientSession()
 
+        if "/enduser-mobile-web/1/enduserAPI/" in server.endpoint:
+            self._is_local = True
+            self._access_token = token
+
+            # To avoid security issue, add the following authority to
+            # your HTTPS client trust store: https://ca.overkiz.com/overkiz-root-ca-2048.crt
+            self._ssl_context = ssl.create_default_context(
+                cafile=os.path.dirname(os.path.realpath(__file__))
+                + "/overkiz-root-ca-2048.crt"
+            )
+            self._ssl_context = None
```

Comment:
Blocked by https://github.com/Somfy-Developer/Somfy-TaHoma-Developer-Mode/issues/4

Reply:
SSL check removed for now. We can just pass a session that does not verify SSL.